src/lua/protobuf_wireformat.lua

```diff
@@ -106,6 +173,21 @@ local function encode_tag(field_id, wire_type)
     return encode_varint(bit.bor(bit.lshift(field_id, 3), wire_type))
 end
 
+-- Decode a tag byte.
+local function decode_tag(ctx)
```

Comment:
The module is splitted to the helpers and the public API sections. Seems like the second section contains the function that are exposed by the module.

The `decode_tag` and `set_default` functions are placed into the helpers section, but exposed by the module.

It feels confusing.

Reply:
I agree it will be better to move this functions in API functions. Also it allowed to get rid of forward declaration of `decode_int` function.

Reply:
Seems solved.